frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -150,3 +143,12 @@ export const menuActions = [
   Kebab.factory.ModifyAnnotations,
   Kebab.factory.Delete,
 ];
+
+export const menuActionsCreator = (kindObj: K8sKind, vm: VMKind, { vmi, pods, migrations }) => {
```

Comment:
Do we really need to export both `menuActions` and `menuActionsCreator`?

It seems that `menuActionsCreator` is the one who is supposed to interpret (execute) `menuActions`.

Reply:
yes we do. The use cases differ:
- **vm list**: the row already has status, etc computed so it just passes it to `menuActions`
- **vm detail**:  Once we get detail page resources we compute the status, etc and pass it to all the actions (which is the only way how to pass logic to execute after loading resources before rendering actions)